src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,23 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+import classNames from 'classnames';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  return (
+    <article
+      data-id={todo.id}
+      className={classNames('TodoInfo', {
+        'TodoInfo--completed': todo.completed,
+      })}
+    >
+      <h2 className="TodoInfo__title">{todo.title}</h2>
+
+      <UserInfo user={todo.user} />
```

Comment:
render this component only when user exists
```suggestion
      {todo.user && <UserInfo user={todo.user} />}
```